### DISCUSSION TITLE
Visibility of user icons/pins

### DISCUSSION THREAD

**Discussion by pwb999** [organicmaps/organicmaps#2125] [Ideas]
Great work Organic Team. Keep it up..

I have added lots of pins and linestrings to OM. I note in the KML file there is a visibility tag. Does this do anything as it would be great to have the different pins display on different Zoom levels.
Also I use my own UK Official Contours kml file which is more detailed than the standard NASA STRM one. Can I turn the NASA one off or hid it somehow.

Best wishes
Pete

**Comment by biodranik** [comment]
1. What type of logic can be implemented about visibility? The current tag stores the whole bookmark list's visibility setting (you can hide the whole list from the bookmarks dialog).
2. There is a button to turn off elevation info, either in the bottom-right menu or at the top left (layers) button.

**Reply by pwb999** [reply]
Hi,
For example I have a 10,000 set of pins denoting Summits. 
If I set the visibility to 14 they still ALL appear at normal zoom level. I want to only show these summits when I am fully zoomed into that area of the screen. Is this possible??
  
<name>VIS14</name>
  <visibility>14</visibility>
  <ExtendedData xmlns:mwm="https://omaps.app">

**Reply by biodranik** [reply]
In theory, everything is possible :) But it is not clear how to properly implement it. Create some specific setting/property of a bookmark list, where you can set the zoom level?

Regarding summits, wouldn't it be better to contribute missing summits into OpenStreetMap so everyone will enjoy seeing them? Also, the OM search can find them properly.
What additional information is stored in your bookmarks that are absent in OSM?

**Comment by pwb999** [comment]
Brits love their hills!!!

Please see https://www.hill-bagging.co.uk for the sort of information that is collected freely by hikers etc.
There are about 30 fields for EACH Hill so not really possible to upload direct to OSM. I do update OSM on my walks and have done so for 15 years or so. This image is just a bit of the database which is free to download for all ...
As you can see there are many categories of hills and it would be neat to filter the hills into the one type you are interested in. Most by the way are very very small by EU standards. Still we love them!!

<img width="586" alt="Screenshot 2022-02-18 at 20 04 50" src="https://user-images.githubusercontent.com/58628197/154754115-47b8ed55-45a0-4ac1-85f8-8910abfb4dfa.png">
